src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/jpa/VisitId.kt

```diff
@@ -19,20 +16,30 @@ data class VisitId(
    */
   val label: String? = null,
 
-  @Enumerated(EnumType.STRING)
   val mappingType: MappingType,
 
-) {
+  @Transient
+  @Value("false")
+  val new: Boolean = true,
```

Comment:
this is a bit naff, but with r2dbc when doing a `save` we have to specify if the entity is new or existing if the `id` is set